build/bin/build-server-rpm

```diff
@@ -30,7 +30,7 @@ is_build_needed() {
         s3://pmm-build-cache/${spec_name}-${rpm_version} \
         ${rpms_dir}/${spec_name}-${rpm_version}
 
-    local packages=$(find ${rpms_dir}/${spec_name}-${rpm_version} -name "*.rpm" | wc -l)
+    local packages=$(find ${rpms_dir}/${spec_name}-${rpm_version} -name "*.rpm" | grep -v "pmm-server-" | wc -l)
```

Comment:
There should be comment explains why we skip pmm-server.

Reply:
It has been added.